src/System.Collections.Concurrent/ref/System.Collections.Concurrent.cs

```diff
@@ -64,6 +64,9 @@ public partial class ConcurrentBag<T> : System.Collections.Concurrent.IProducerC
         bool System.Collections.ICollection.IsSynchronized { get { throw null; } }
         object System.Collections.ICollection.SyncRoot { get { throw null; } }
         public void Add(T item) { }
+#if netcoreapp11
```

Comment:
@joperezr is in the process of updating the ref projects to only have netcoreapp and no netstandard configs any more.